codespell_lib/tests/test_dictionary.py

```diff
@@ -65,17 +68,24 @@ def test_dictionary_formatting(fname, in_aspell):
 
 
 def _check_aspell(phrase, msg, in_aspell, fname):
-    if speller is None:
+    if not spellers:  # if no spellcheckers exist
         return  # cannot check
     if in_aspell is None:
         return  # don't check
+    elif in_aspell is False:
+        language = supported_languages
+    elif in_aspell is True:
+        language = ('en',)
```

Comment:
Should a True in in_aspell lead to a check in en alone or a check in all dictionaries. I would favor the second option.